faunadb-common/src/main/java/com/faunadb/common/RefAwareHttpClient.java

```diff
@@ -29,13 +28,13 @@ public void close() {
       if (refCount.decrementAndGet() < INITIAL_REF_COUNT && !delegate.isClosed()) {
         delegate.close();
       }
-    } catch (IOException e) {
+    } catch (Exception e) {
```

Comment:
Why did we change this to `Exception`? The code becomes more fragile with this change.